### DISCUSSION TITLE
GO Closes on Start

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#540] [Help]
Originally posted 2019-01-26 18:56:31.554000 by drdarin in GrandOrgue discussion forum on SourceForge<br><br>I simply changed the audio output from Direct Sound USB Headphones to Direct Sound Realtek Audio and the app closes while loading any instrument, even the demo. First, it would only play with repeated delay, then, after closing and reopening, it closes before I can even see an instrument image.

I&#039;m using the latest version of the GO installer for Win64 - GrandOrgue-0.3.1.2313-154.1-win64.exe

**Comment by ExBaker** [comment]
Hi Lars!

I started having the exact same issue two days ago. GO, Windows 11, and my audio interface driver are all up to date. GO closes immediately on launch. Through trial and error, I was able to launch GO correctly if my interface (Audient id44 MK1) was turned OFF. GO then issued the error that there would be no sound, as expected. If I then turned ON my interface and selected it in RTaudio:asio mode, I could load organs and play them successfully. If I then closed GO and relaunched it with the interface still ON, it will immediately crash to the desktop.

I captured this from the Windows Event Viewer. It's definitely a crash in GO.

Faulting application name: GrandOrgue.exe, version: 3.13.3.0, time stamp: 0x00000000
Faulting module name: GrandOrgue.exe, version: 3.13.3.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x000000000025b425
Faulting process id: 0x0xFF0
Faulting application start time: 0x0x1DA6460865D3EB9
Faulting application path: C:\Program Files\GrandOrgue\bin\GrandOrgue.exe
Faulting module path: C:\Program Files\GrandOrgue\bin\GrandOrgue.exe
Report Id: f87a0989-9aac-4dcc-b685-18ce519d43b1
Faulting package full name:
Faulting package-relative application ID:​

Any ideas why it doesn't seem to want to start?

TIA,
Tom

**Comment by midimusic** [comment]
You appear to be using asio - this has no benefits and is not needed on windows and often causes trouble. I suggest you uninstall it and remove all traces of it on your computer. Then try agian.
csw900

**Comment by ExBaker** [comment]
Thanks for replying. In my 30 years of experience with digital audio workstations, ASIO has been the gold standard in audio drivers. It provides the lowest latency in the Windows environment and is extremely stable and reliable. I'm using the native ASIO driver for my audio interface written by the manufacturer and can achieve 0.7ms latency in my DAW.Perhaps you are thinking of ASIO4ALL?  That, I agree, is unnecessary, as it's not a true ASIO driver, but merely a wrapper around a WDM kernel. Best,Tom
-------- Original message --------From: midimusic ***@***.***> Date: 2/23/24  3:41 AM  (GMT-05:00) To: GrandOrgue/grandorgue ***@***.***> Cc: ExBaker ***@***.***>, Comment ***@***.***> Subject: Re: [GrandOrgue/grandorgue] GO Closes on Start (Discussion #540) 
You appear to be using asio - this has no benefits and is not needed on windows and often causes trouble. I suggest you uninstall it and remove all traces of it on your computer. Then try agian.
csw900

**Comment by midimusic** [comment]
Hi Tom

"ASIO has been the gold standard". Maybe your experience goes back too far and you have not kept up to date. Windows no longer needs ASIO - that is a fact. Check for yourself. I think full details are on a Microsoft website (from memory).

"can achieve 0.7ms latency". Have you MEASURED the real latency of your organ? If you do so you are likely to find that it is around 70 to 100ms on Windows. Thus your 0.7ms is not relevant.

I still think that ASIO could be the cause of your problem - until proved otherwise. Especially if your organ works correctly with the standard HD audio of your computer - that is with your interface card unplugged.

Best Wishes csw900

**Comment by ExBaker** [comment]
The issue I reported has been resolved by the GO programmers. Apparently, the most recent ASIO SDK included a change that necessitated a change to the GO code.
